autopush/jwt.py

```diff
@@ -78,6 +78,20 @@ def extract_signature(auth):
         return payload, encoded
 
     @staticmethod
+    def extract(token):
```

Comment:
Maybe rename to ``extract_assertion`` ?

On first glance why you'd use decode vs. extract makes little sense without reading docs. Having slightly more info in the method name would help readability. Maybe they should both be ``decode_*`` for consistency, or ``extract_*``?